api/room/schema.py

```diff
@@ -93,8 +95,15 @@ def mutate(self, info, **kwargs):
         if kwargs.get('room_tags'):
             room_tags = kwargs.pop('room_tags')
         room = RoomModel(**kwargs)
-        save_room_tags(room, room_tags)
-        return CreateRoom(room=room)
+        payload = {
```

Comment:
What happens when the user fails to provide `structure_id`?

Reply:
The user gets an error message telling them that the `structure_id ` is needed.